review.js

```diff
@@ -0,0 +1,132 @@
+import "./style.css";
+// SELECT REQUIRED ELEMENTS
+const todoInput = document.querySelector("#addlist");
+const todoForm = document.querySelector(".todo-form");
+const todoWrapper = document.querySelector(".wrapper");
+const deleteCompleted = document.querySelector(".remove");
+const addLocalStorage = (todo) => {
+  const todos = JSON.parse(localStorage.getItem("todos")) || [];
+  todos.push(todo);
+  localStorage.setItem("todos", JSON.stringify(todos));
+};
+
+// ADD TODO
+const addTodo = (e) => {
```

Comment:
As an optional suggestion
-Make sure to use a descriptive name for the event objects rather than a simple (e) letter.
-It is a best practice to keep Javascript functions in separate files because of two reasons
 1. Separation of concerns - by having the code in a separate file it is separate from the HTML and may be changed without touching the HTML file. If used in multiple places it allows for a single place to make the change.

2. DRY - Don't Repeat Yourself By having it in a separate file you don't have to repeat the code in several HTML files.
 
so considering the above reasons kindly adjust you js functions in a separate script file.